src/modules/Helpers/Safebox.js

```diff
@@ -73,8 +73,7 @@ export default class Safebox extends BaseHelper {
     result.double_encryption = profile.doubleEncryption.value;
     result.reply_enabled = profile.replyEnabled.value;
     result.notification_language = this.notificationLanguage || 'en';
-    result.user_email = this.userEmail;
```

Comment:
So in the end, the user email parameter was part of the request, but only as an attribute of the safebox, and not as a top level attribute?

In the case, I am wondering if the fix should be made server side instead. I will talk with Laurence about it.